internal/observability/exporter/jaeger/jaeger.go

```diff
@@ -29,7 +29,6 @@ type jaegerOptions = jaeger.Options
 type Jaeger interface {
 	Start(ctx context.Context) error
 	Stop(ctx context.Context)
```

Comment:
import root interface package and change below
```suggestion
	exporter.Exporter
```